src/en/space-station-14/round-flow/proposals/specter.md

```diff
@@ -0,0 +1,151 @@
+However, they are able to read both normal and deadchat.
+
+They are able to see the fear level of player's through status icons.
+The terrified and hysterical fear levels are marked with a slow and fast flashing respectively, in order to signal when objectives can be completed or souls can be claimed.
+
+There are two primary resources that the Specter must manage: _power_ and _souls_.
+
+_Power_ is the Specter's resource pool for a few different things.
+Power is used both for activating abilities and as the Specter's health.
+
+_Terror_ is used by the Specter to unlock abilities that are already revealed.
+It is accumulated alongside Power when the Specter is nearby players with high amounts of fear.
+Using abilities causes the specter to be briefly visible but still unable to take damage.
+However, this allows people to throw salt or use the bible on them.
+
+_Souls_ are the Specter's form of progression.
+A soul is gained every time a _haunt_ is successfully completed.
+At 3, 6, and 9 souls, the Specter gains more powerful abilities that can be unlocked.
+
+The Specter is primarily counted through the use of holy items: salt and the Bible.
+Stepping on salt forces the Specter into the "haunt" mode, making them vulnerable.
+Using the bible on the Specter after it uses an ability causes the same effect.
+
+### Haunting
+
+Haunting is the Specter's main attack as well as the time when it's most vulnerable. 
+A haunt can be started at any time as long as the Specter isn't standing in a wall and there are no players in line of sight.
+
+When a haunt begins, the Specter becomes visible to everyone, loses access to their abilities, and can be hurt and killed.
+While active, however, the Specter causes fear to build up for anyone nearby.
+
+While in the haunt form, the Specter gains power and terror based on the fear of nearby players.
+This pushes the Specter to regularly haunt while stalking a player in order to replenish their Power.
+
+At any point, if the Specter gets close to someone whose fear is at the Hysterical level, the Specter can click on them to claim their soul.
+This instantly kills the victim, rewards the Specter with a soul, and increases their max power by 50.
+
+The haunt lasts for 60 seconds or until a soul is claimed.
+Afterwards, there is a minute-long cooldown before another haunt can start.
+
+## Objectives
+- Claim 7-12 souls
+- Raise 15-20 players' fear to Hysterical
+- Raise 3-5 player's fear to Terrified or higher simultaneously
+- Keep someone's fear at Terrified or higher for 3 minutes
+- Generate 3000 power through fear (excludes baseline power generation rate)
+
+## Abilities
+
+#### Tier 0 Abilities 
+Available roundstart at 0 souls.
+
+- **Whisper:**: Sends a message to a target player, slightly increasing their fear.
+- **Ectoplasm:** Secretes ectoplasm around the Specter in a small radius.
+Slipping on the ectoplasm increases fear and stuns like soap.
```

Comment:
is this limited use or infinite, you could have a spooky slip n slide that farms fear and tell everyone to have "fun" if its infinite

Reply:
you can just clean it up and put it in a garbage bag. if the entire station chooses to slip and slide like idiots that's on them.